README.md

```diff
@@ -400,6 +402,16 @@ Required for [Docker](#docker) projects.
 
 Required for [balena](#balena) projects.
 
+#### `PYPI_TOKEN`
```

Comment:
missing newline for Markdown syntax (try a markdown lint vscode extension)

Reply:
done